### PR TITLE
[v2] Clean up 

### PR DESCRIPTION
**Pull request by emmabukacek**
## Changes
* Sets tabs to spaces in `README.md`
* Remove unnecessary snippet annotations in `README.md`
* Use `Object.assign` in core `init` and remove `TODO` comment

getinsertpic.com

### CODE REVIEW

File: src/core/init.js [hunk 1]
```diff
@@ -16,8 +14,7 @@ const defaultOptions = {
  * @returns {Function} Always returns a function, determined by cli flags.
  */
 const init = function(options, pathPassed) {
-  // TODO: Object.assign
-  const optionsWithDefaults = defaults(options || {}, defaultOptions);
+  const optionsWithDefaults = Object.assign(options || {}, defaultOptions);
```
Comment (SimenB) [inline src/core/init.js]:
```js
const optionsWithDefaults = Object.assign({}, options, defaultOptions);
```

**Comment by SimenB** [comment]
@emmabukacek this now has some conflicts as I merged your other PR. Care to rebase?

Also, would you be interested in being a maintainer? I've spent an embarrassingly small amount of time on this project the last months, and some help would be greatly appreciated!

File: src/core/init.js (hunk 1 above)
Comment (emmabukacek) [reply src/core/init.js]: Oh duh, haha. Thanks!

File: src/utils/getFormatter.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ const NATIVE_FORMATTER_PATH = path.join('src', 'formatters');
  */
 const loadFormatter = function(formatterPath) {
   try {
-    /* eslint-disable import/no-dynamic-require */
+    // eslint-disable import/no-dynamic-require
```
Comment (SimenB) [inline src/utils/getFormatter.js]: `// eslint-disable-next-line  import/no-dynamic-require` allows you to delete the whole `// eslint-enable` line

**Comment by emmabukacek** [comment]
@SimenB Hey! Think this is ready for another look.

As for being a maintainer, I'm definitely interested! I sent an email over to ya.

**Pull request merged**